frontend/src/views/WheresMyGene/components/Filters/index.tsx

```diff
@@ -71,10 +72,15 @@ const filterOptions = createFilterOptions({
     `${option.label} ${option.collection_label}`,
 });
 
+function isOptionEqualToValue(option: FilterOption, value: FilterOption) {
+  return option.id === value.id;
```

Comment:
just curious, by default it's matching `name`, how did that cause double selection? Thanks!